polyglot/piranha/Cargo.toml

```diff
@@ -44,8 +44,8 @@ tree-sitter-java = { git = "https://github.com/tree-sitter/tree-sitter-java.git"
 tree-sitter-swift = { git = "https://github.com/ketkarameya/tree-sitter-swift.git", branch = "add_parser" }
 tree-sitter-strings = { git = "https://github.com/ketkarameya/tree-sitter-strings.git" }
 tree-sitter-python = { git = "https://github.com/tree-sitter/tree-sitter-python.git" }
-tree-sitter-go = { git = "https://github.com/tree-sitter/tree-sitter-go.git" }
 tree-sitter-typescript = { git = "https://github.com/tree-sitter/tree-sitter-typescript.git" }
+tree-sitter-go = { git = "https://github.com/dvmarcilio/tree-sitter-go.git", rev = "0c63030" }
```

Comment:
Please update this to the uber fork

Reply:
@ketkarameya we can move it once https://github.com/uber/tree-sitter-go/pull/1 gets approved.

Reply:
awesome.

Reply:
https://github.com/uber/tree-sitter-go/pull/2 now!